src/contracts/middleware/VoteWeigherBase.sol

```diff
@@ -59,29 +59,25 @@ contract VoteWeigherBase is VoteWeigherBaseStorage {
      * @notice This function computes the total weight of the @param operator in the quorum @param quorumNumber.
      * @dev returns zero in the case that `quorumNumber` is greater than or equal to `quorumCount`
      */
-    function weightOfOperator(uint8 quorumNumber, address operator) public virtual returns (uint96) {
+    function weightOfOperator(uint8 quorumNumber, address operator) public virtual validQuorumNumber(quorumNumber) returns (uint96) {
```

Comment:
the dev tag is now outdated with this change. the function now *reverts* rather than returning 0

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/81/commits/7d71e880e8adaaeca4a5b8a919abcbb852610fa7